### DISCUSSION TITLE
SilentArgs isn't working to install Git-lfs for v3.3.0

### DISCUSSION THREAD

**Discussion by Shiva20232023** [git-lfs/git-lfs#5287] [Q&A]
![error](https://user-images.githubusercontent.com/125924558/220199222-62d27926-7557-46cc-9e69-6ff2909fd4b7.jpg)
 
SilentArgs isn't working to install Git-lfs for v3.3.0,

Fails to install V3.3.0, because when we try to install .exe file manually it's popping a question (PFA), as a result i'm unable to install Git-LFS using SilentArgs !

suppress that question might help.

Appreciate your help.

Shiva.

**Comment by bk2204** [comment]
Hey,

This question is required for security, since the install process invokes Git.  Unfortunately, many Windows systems have insecure PATH locations including the current directory or globally writable directories, and as such, we have to verify a nonstandard PATH with the user.

If you install Git in the normal, default PATH, then this question won't be posed.  Alternatively, you can use the standalone binary for your system, which is relocatable and doesn't invoke an installer.

**Reply by Shiva20232023** [reply]
So currently, there is no method to explicitly suppress the security question for silent installs?

FYI, we install Git on a separate presented drive for applications on our automation agent servers.

**Reply by bk2204** [reply]
That's correct.  If you want a non-interactive install, you want to install it using the regular binary, not the installer.

I'll point out that since none of the core team members are Windows users, the Windows installer is completely best effort.
